packages/cli/test/unit/commands/build/index.test.ts

```diff
@@ -1250,6 +1250,24 @@ describe('build', () => {
       (await fs.readFile(join(output, 'static/index.txt'), 'utf8')).trim()
     ).toEqual('marketing');
   });
+
+  it('should write to flags.json', async () => {
+    const cwd = fixture('with-flags'); // TODO
+    const output = join(cwd, '.vercel', 'output');
+
+    const exitCode = await build(client);
+    expect(exitCode).toEqual(0);
+
+    // TODO: Check the content of `flags.json`
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏remove?

This is inspected now, right?